builder/source/simple.go

```diff
@@ -112,50 +113,52 @@ func (s *SimpleSource) IsFetched() bool {
 	return PathExists(s.GetPath(s.validator))
 }
 
-// download utilises CURL to do all downloads
+// download downloads simple files using go grab
 func (s *SimpleSource) download(destination string) error {
-	hnd := curl.EasyInit()
-	defer hnd.Cleanup()
 
-	hnd.Setopt(curl.OPT_URL, s.URI)
-	hnd.Setopt(curl.OPT_FOLLOWLOCATION, 1)
+	file, err := os.Create(destination)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
 
-	out, err := os.Create(destination)
+	client := grab.NewClient()
+	req, err := grab.NewRequest(destination, s.URI)
 	if err != nil {
 		return err
 	}
 
-	pbar := pb.New64(0)
-	pbar.Set(pb.Bytes, true)
-	pbar.Set("prefix", filepath.Base(destination))
-	pbar.SetMaxWidth(80)
+	resp := client.Do(req)
 
-	writer := func(data []byte, udata interface{}) bool {
-		if _, err := out.Write(data); err != nil {
-			return false
+	// Setup our progress bar
+	var pbar *pb.ProgressBar
+	pbar = pb.Start64(resp.Size())
+	pbar.Set(pb.Bytes, true)
+	// Default template except that the 'per second' suffix is changed to '/s' instead of 'p/s'
+	progressBarTemplate := `{{with string . "prefix"}}{{.}} {{end}}{{counters . }} {{bar . }} {{percent . }} {{speed . "%s/s" "??/s"}}{{with string . "suffix"}} {{.}}{{end}}`
```

Comment:
I don't agree here. We're likely not reusing this template anywhere else. Instead, maybe we can just pass the literal string instead of defining a dedicated variable.

Reply:
The main reason for this suggestion is actually not for re-use, but to make the function easier to read by getting this huge string (and the accompanying comment) out of the way.